src/main/webapp/js/common.js

```diff
@@ -441,11 +441,11 @@ function setStatusMessage(message, error) {
         $(DIV_STATUS_MESSAGE).attr('class', 'alert alert-warning');
     }
 
-    Element.prototype.documentOffsetTop = function() {
-        return this.offsetTop + (this.offsetParent ? this.offsetParent.documentOffsetTop() : 0);
+    function getDocumentOffsetTop(element) {
```

Comment:
Actually, this should be `getElementOffsetTopRelativeToDocument`.
Offset is an HTML lingo for the distance of the current element relative to another (usually its parent element). Should I add a comment to explain that?